### PR TITLE
Update chains.ts

### PR DESCRIPTION
**Pull request by Kweiss**
Adding PGN

**Description**

A clear and concise description of the features you're adding in this pull request.

**Tests**

Please describe any tests you've added. If you've added no tests, or left important behavior untested, please explain why not.

**Additional context**

Add any other context about the problem you're solving.

**Metadata**

- Fixes #[Link to Issue]

### CODE REVIEW

**Comment by hamdiallam** [comment]
@Kweiss the test script failed since pgn needs to be added as a supported chain in the `src/types.ts` file

File: src/chains.ts [hunk 1]
```diff
@@ -25,12 +25,26 @@ export const NETWORK_DATA: Record<Chain, Network> = {
...
   goerli: {
     id: 5,
     name: 'Goerli',
     provider: new ethers.providers.InfuraProvider('goerli'),
     layer: 1,
   },
+  sepolia: {
+    id: 11155111,
+    name: 'Sepolia',
+    provider: new ethers.providers.InfuraProvider('goerli'),
```
Comment (annieke) [inline src/chains.ts:45]: this should probably be sepolia?

**Review by annieke — DISMISSED** [review]
this add to [`src/types.ts`](https://github.com/ethereum-optimism/ethereum-optimism.github.io/blob/master/src/types.ts) please!

**Review by roninjin10 — APPROVED** [review]

**Review by roninjin10 — DISMISSED** [review]
A type is not updated run tsc and tests locally and it will point you to the right spot

File: src/chains.ts [hunk 2]
```diff
@@ -25,12 +25,26 @@ export const NETWORK_DATA: Record<Chain, Network> = {
     ),
     layer: 2,
   },
+  pgn: {
+    id: 424,
+    name: 'PGN - Public Goods Network',
+    provider: new ethers.providers.StaticJsonRpcProvider(
+      'https://rps.publicgoods.network'
```
Comment (hamdiallam) [inline src/chains.ts:32]: I believe there's a typo here /s/rps/rpc

**Review by hamdiallam — DISMISSED** [review]

File: src/chains.ts (hunk 2 above)
Comment (annieke) [reply src/chains.ts:32]: @Kweiss ? :)

**Pull request merged**